readthedocs/projects/tasks/builds.py

```diff
@@ -508,6 +508,10 @@ def on_failure(self, exc, task_id, args, kwargs, einfo):
                 build_pk=self.data.build['id'],
                 commit=self.data.build_commit,
                 status=status,
+                # Fix bug where we were generating relative links to the version
+                # with ``get_absolute_url``.
+                # https://github.com/readthedocs/readthedocs.org/issues/9791
```

Comment:
While writing my last comment, I found the `get_absolute_url` is not the problem. It returns full URL "sometimes". Please, read the final comment.

The comment from the code can just be removed since it's not accurate.

Reply:
@humitos I don't fully understand this comment, can you edit your comment to just have the part that I should address? :)

Reply:
Just read https://github.com/readthedocs/readthedocs.org/pull/9807#pullrequestreview-1217124674 which has all the context here 😄

Reply:
I should disregard everything after `Finally, this part of the workflow seems to be broken`, right?

Reply:
Yes